lang/ilias_de.lang

```diff
@@ -13789,7 +13789,7 @@ ecs#:#ecs_member_auth_type#:#Authentifizierungsmethode für Kursmitglieder
 background_tasks#:#background_tasks#:#Background Tasks
 background_tasks#:#background_tasks_running#:#Background Tasks
 background_tasks#:#scheduled#:#Geplant
-background_tasks#:#dismiss#:#Schliessen
+background_tasks#:#dismiss#:#Entfernen
```

Comment:
@d3r1w could you please rename the internal variable from `dismiss` to `remove` as well? thanks!

Reply:
@chfsx should I only change the langvar or also the task interface and the abstract task constants? According to 45c6fc580501c08e5162b03d11063090ef85f672.

Reply:
Yay for consistency. Thx!